dataflow/manual/content.tex

```diff
@@ -299,6 +299,31 @@ \subsubsection{AbstractValue}
 class NullnessValue extends CFAbstractValue<NullnessValue>
 \end{verbatim}
 
+\subsubsection{UnusedAbstractValue}
+\label{sec:UnusedAbstractValue_classes}
+
+UnusedAbstractValue, as the name suggests, is an AbstractValue but is
+not involved in any computation during dataflow analysis. That is, the
+UnusedAbstractValue implements the leastUpperBound method in a way it
+just throws an error if it is called, without computing the lub with the
+other AbstractValue.
+
+It can be used in an analysis which does not require any computation
```

Comment:
You can mention that, in some situations, lub can be computed on the store level, followed by the example you have.

Reply:
Done.